platforms/common/uORB/uORBManager.cpp

```diff
@@ -491,14 +493,13 @@ uORB::Manager::callback_thread(int argc, char *argv[])
 	while (true) {
 		lockThread(per_process_lock);
 
-		class SubscriptionCallback *sub = dequeueCallback(per_process_lock);
```

Comment:
This now uses / assumes the kernel side callback_thread is a pthread, is it ? (It's not)

Reply:
ahh. the mutex should be px4_sem. will change